internal/proxy/options.go

```diff
@@ -71,6 +72,8 @@ type Options struct {
 	CookieSecure   bool          `envconfig:"COOKIE_SECURE" default:"true"`
 	CookieHTTPOnly bool          `envconfig:"COOKIE_HTTP_ONLY"`
 
+	PassAccessToken bool `envconfig:"PASS_ACCESS_TOKEN" default:"false"`
```

Comment:
can we add a simple unit test here for the new config value in `options_test.go`?

Reply:
And done as well!